extensions/amp-accordion/amp-accordion.md

```diff
@@ -267,6 +267,18 @@ it collapses all sections of the accordion. To specify a section, add the
 </button>
 ```
 
+### Actions in Bento Mode
+
+In Bento mode, the above actions can be called via `getApi()`. For example:
+
+```html
+<script>
```

Comment:
I'd rather not have `<script>` or `querySelector`. But simply do:

```
const api = await element.getApy();
api.toggle(...);
```

Reply:
I think it'd be good to have the HTML element definition in the snippet as well. This way we have a complete and functional example that is easy to reproduce.

Reply:
+1

Reply:
Here is a more comprehensive example.  We don't have to use something this complex of course, but just wanted to provide an example:

```
<amp-accordion id="myAccordion" animate='true'>
  <section id="section1">
    <h2>Section 1</h2>
    <div>Puppies are cute.</div>
  </section>
  <section id="section2">
    <h2>Section 2</h2>
    <div>Kittens are furry.</div>
  </section>
  <section id="section3">
    <h2>Section 3</h2>
    <div>Elephants have great memory.</div>
  </section>
</amp-accordion>

<div class="buttons" style="margin-top: 8px;">
  <button id='button1'>Toggle All Sections</button>
  <button id='button2'>Expand Section 2</button>
  <button id='button3'>Collapse Section 3</button>
</div>

<script>
  (async () => {
    const accordion = document.querySelector('#myAccordion');
    await customElements.whenDefined('amp-accordion');
    const api = await accordion.getApi();
    
    // programatically toggle (expands) all sections
    api.toggle();

    // set up button actions
    document.querySelector('#button1').onclick = () => api.toggle();
    document.querySelector('#button2').onclick = () => api.expand('section2');
    document.querySelector('#button3').onclick = () => api.collapse('section3');
  })();
</script>
```

Reply:
What would be next steps to have a centralized page with a generic explanation and example that other component pages can point to?

Reply:
I've updated the Bento guide to be more descriptive on accessing Bento component APIs - but I think each component should include a code example that works when copy/pasted. This makes it easier for developers to try out Bento components and takes the guesswork out of implementation details.